opacus/layers/dp_lstm.py

```diff
@@ -301,8 +400,20 @@ def forward(
             - ``c_n`` is of shape ``[L * P, B, H]`` and contains the cell state for ``t = T``.
         """
 
-        x = self._rearrange_batch_dim(x)
-        T, B, D = x.shape
+        if isinstance(x, PackedSequence):
+            x, batch_sizes, sorted_indices, unsorted_indices = x
```

Comment:
No reason to "unpack" the sequence here I think. Just send it down as-is, and let the actual class that needs to work with it handle it